src/GraphQL.Client/GraphQLHttpClientOptions.cs

```diff
@@ -87,4 +87,11 @@ public class GraphQLHttpClientOptions
     /// </summary>
     public ProductInfoHeaderValue? DefaultUserAgentRequestHeader { get; set; }
         = new ProductInfoHeaderValue(typeof(GraphQLHttpClient).Assembly.GetName().Name, typeof(GraphQLHttpClient).Assembly.GetName().Version.ToString());
+
+    /// <summary>
+    /// Delegate permitting use of <see href="https://www.apollographql.com/docs/react/api/link/persisted-queries/">Automatic Persisted Queries (APQ)</see>.
+    /// By default returns <see langword="true" /> for all requests. Note that GraphQL server should support APQ. Otherwise, the client disables APQ completely
+    /// after an unsuccessful attempt to send an APQ request and then send only regular requests.
+    /// </summary>
+    public Func<GraphQLRequest, bool> EnableAutomaticPersistedQueries { get; set; } = _ => true;
```

Comment:
Don't know if I would personally default it to enabled

Reply:
The idea here to use APQ if server is ready for it. If not - client fallbacks to regular call for all subsequent requests.

Reply:
Ok, I'll change to false.

Reply:
done